rfc/cli-api.md

```diff
@@ -0,0 +1,33 @@
+# CLI compatibility information
+
+The CLI API provides information about the compatibility of the Constellation CLI and other components of the Constellation ecosystem such as Kubernetes versions.
+
+## CLI API Endpoints
+
+The build pipeline produces lookup tables for compatibility information that are uploaded to S3 and can be accessed via HTTP.
+The lookup tables are organized in a directory structure that allows to look up the compatibility for a given Constellation version.
+
+The following HTTP endpoints are available:
+
+- `GET /constellation/v1/ref/<REF>/stream/<STREAM>/<VERSION>/cli/`
+  - [`info.json` returns the CLI compatibility lookup table for the given Constellation version.](#cli-lookup-table)
+
+## CLI lookup table
```

Comment:
The term "lookup table" is somewhat ambiguous in our API specs and daily usage (I know the image-api rfc still uses it). I would prefer something like "info artifact" to differentiate this from the lookup table design we use in the (tighter) versions API.

Reply:
another idea: "cli info".